libaesnd/dspcode/dspmixer.s

```diff
@@ -157,7 +157,8 @@ recv_cmd:
 	cmpi	$acc1.m,#0x0100
 	jeq		send_samples
 	
-	cmpi	$acc1.m,#0xdead
```

Comment:
could you add a comment explaining why this is different from the others? don't make it long, just explain its sign extending